src/components/cardbuilder/cardBuilder.js

```diff
@@ -701,11 +723,15 @@ import 'programStyles';
                     maxWidth: width,
                     tag: item.ParentBackdropImageTags[0]
                 });
+                imgtag = item.ParentBackdropImageTags[0];
 
             }
 
+            blurhash = imageLoader.getImageBlurhashStr(blurhashimg, imgtag);
+
             return {
                 imgUrl: imgUrl,
+                blurhash: blurhash,
```

Comment:
```suggestion
            return {
                imgUrl: imgUrl,
                blurhash: imageLoader.getImageBlurhashStr(blurhashimg, imgtag);,
```

You can do this directly, since the blurhash variable isn't used anywhere else.

Reply:
I guess it's cleaner? But good point.